index.html

```diff
@@ -1177,6 +1179,13 @@
             sync (unless <a>media mirroring</a> is used).
           </div>
           <div class="note">
+            The user agent is encouraged to pass locale information to the
+            <a>remote playback device</a> when possible, so that the <a>remote
+            playback device</a> may adjust its user interface and operational
```

Comment:
It might be worth describing how this interacts with default text track selection.  For example the remote playback device should choose the same default text track as the controlling user agent (given that user agent's locale preferences are sent from the latter to the former).  Perhaps this is implied by your comment already though.

Reply:
It was meant to be implied, but it seems worth calling out one or two examples of why that information is useful for remote playback devices. I added a sentence of default text track selection, and a note on the HTTP Accept-Language header.